examples/advanced/custom-formats-with-templates/README.md

```diff
@@ -50,7 +50,7 @@ Now open the `build.js` script and look at how these custom formats are declared
 Finally, look at the different template files in the `templates` folder and see how they are built to generate custom file formats in output:
 
 * **web-scss.template**: this is a template that uses [Lodash](https://lodash.com/docs/4.17.10#template), and shows how you can create a custom format for Sass ".scss" files. The same approach can be used if you need other custom formats for the web (or other platforms too).
-* **android-xml.template**: this template too uses Lodash, and shows how to create a custom XML format for Android, that can be read as "resource file". This is just one of the many possible formats for Android, so if you need to create one speak with your developers to agree on the format they want.
+* **android-xml.template**: this template too uses Lodash, and shows how to create a custom XML format for Android, that can be read as "resource file". This is one of the many possible formats for Android, if you need to create one speak with your developers to agree on the format they want.
```

Comment:
this sounds funny, should be reworded or cleaned up with appropriate punctuation

Reply:
@chazzmoney : Kindly confirm on below statement, added punctuation
This is one of the many possible formats for Android, if you need to create one, speak with your developers to agree on the format they want.